x/attribute/keeper/keeper.go

```diff
@@ -439,10 +450,15 @@ func (k Keeper) PurgeAttribute(ctx sdk.Context, name string, owner sdk.AccAddres
 		return err
 	}
 	for _, acct := range accts {
+		attrToDelete := [][]byte{}
 		store := ctx.KVStore(k.storeKey)
```

Comment:
This should be moved outside the loop. There's no reason to create it again for each account.

Reply:
I thought it made more sense here because I needed to do the `DecAttrNameAddressLookup` call while in the context of having the acct.  Anyway, the point is moot now since I have refactored this area.